src/relayclient/KnownRelaysManager.ts

```diff
@@ -124,6 +124,7 @@ export default class KnownRelaysManager implements IKnownRelaysManager {
     if (this.config.verbose) {
       console.log('fetchRelaysAdded: found unique relays:', foundRelayManagers)
     }
+    if (foundRelayManagers.size === 0) { throw new Error('no registered relayers found') }
```

Comment:
Is it necessary to interrupt the flow here? What is the current behaviour and why is it not intended?

Reply:
ok, removed.

Reply:
ok, looking deeper this will introduce more problems then solve, as 'refresh' will throw even if working with preferred relays only